index.html

```diff
@@ -196,7 +175,32 @@ <h3>OTC Talks #1</h3>
 
 			<h2 id="past-talks">Past Talks</h2>
 
-			<div class="event">We're conducting our first talk soon!</div>
+			<div class="event">
+				<h3>OTC Talks #1</h3>
+				<div>24th December 2022, 10:30 PM IST</div>
+				<p>
+					Linked to OTC CatchUp #111 —
+					<a
+						href="https://catchup.ourtech.community/summary/111"
+						target="_blank"
+						rel="noreferrer"
+						>Summary</a
+					>
+				</p>
```

Comment:
```suggestion
				<p>
					Before
					<a
						href="https://catchup.ourtech.community/summary/111"
						target="_blank"
						rel="noreferrer"
						>OTC CatchUp #111</a
					>
				</p>
```

Reply:
'Held before OTC CatchUp \#111' can be some alternative text...

Reply:
Done.